background.js

```diff
@@ -244,6 +244,19 @@ $(document).ready(function () {
                     chrome.tabs.getSelected(function(tab) { 
                     chrome.bookmarks.create({ 'title': tab.title,'url': tab.url});
                             });
+        }else if (data.result.metadata.intentName == "next_tab") { 
+              var currenttab;
```

Comment:
can you write a function and call that function over here,lets not keep if's more that 3-4 lines. code will get harder to read